src/Components/Header/UserDisplay.js

```diff
@@ -44,7 +44,7 @@ const UserDisplay = () => {
       <div>
         <Signout>
           <button color="primary" onClick={handleClickProfileMenu}>
-            {appUser.name.substring(0, 1)}
+            {appUser.name ? appUser.name.substring(0, 1) : ""}

```

Comment:
This is needed since we don't ask admins to set a users' name anymore. We have a [ticket](https://www.notion.so/uwblueprintexecs/Active-Epics-95f65fd1b589465e9adbe5ead3815e2e?p=bd7c5e3b022840cda39b3e28a0ba84db) to ask the user for their own name in the backlog!